nixos/modules/services/web-servers/darkhttpd.nix

```diff
@@ -1,11 +1,11 @@
 { config, lib, pkgs, ... }:
 
-with lib;
-
 let
+  inherit (lib) mkIf mkOption optional;
+  inherit (lib.types) path bool listOf str port;
   cfg = config.services.darkhttpd;
 
-  args = concatStringsSep " " ([
+  args = lib.concatStringsSep " " ([
     cfg.rootDir
```

Comment:
It's a bit weird to mix both styles; explicit `lib` references aswell as inherits IMHO.

Reply:
I usually go by using inherits only when there is more then two occurences. This has only one, therefore the explicit `lib`. But if you want I can change it.